### PR TITLE
Remove ApiAuthorization.IdentityServer

### PR DESCRIPTION
**Pull request by halter73**
Remove Microsoft.AspNetCore.ApiAuthorization.IdentityServer

#46131

### CODE REVIEW

File: .config/CredScanSuppressions.json [hunk 1]
```diff
@@ -69,22 +69,6 @@
       "file": "\\src\\DefaultBuilder\\test\\Microsoft.AspNetCore.FunctionalTests\\testCert.pfx",
       "_justification": "Legitimate UT certificate file with private key"
     },
-    {
```
Comment (wtgodbe) [inline .config/CredScanSuppressions.json:72]: Thanks for catching these 🥳

File: eng/Baseline.xml [hunk 2]
```diff
@@ -8,7 +8,6 @@ Update this list when preparing for a new patch.
   <Package Id="AspNetCoreRuntime.6.0.x64" Version="6.0.0" />
   <Package Id="AspNetCoreRuntime.6.0.x86" Version="6.0.0" />
   <Package Id="dotnet-sql-cache" Version="6.0.0" />
-  <Package Id="Microsoft.AspNetCore.ApiAuthorization.IdentityServer" Version="6.0.0" />
```
Comment (wtgodbe) [inline eng/Baseline.xml:11]: Undo this, the baseline lists previously released packages, not the current set

File: eng/scripts/CodeCheck.ps1 [hunk 3]
```diff
@@ -248,7 +248,7 @@ try {
             foreach ($file in $changedFilesFromTarget) {
                 # Check for changes in Shipped in all branches
                 if ($file -like '*PublicAPI.Shipped.txt') {
-                    if (!$file.Contains('DevServer/src/PublicAPI.Shipped.txt')) {
+                    if (!$file.Contains('DevServer/src/PublicAPI.Shipped.txt') -and !$file.Contains('ApiAuthorization.IdentityServer/src/PublicAPI.Shipped.txt')) {
```
Comment (wtgodbe) [inline eng/scripts/CodeCheck.ps1:251]: Remove this, I'll just merge on red

**Review by wtgodbe — CHANGES_REQUESTED** [review]
Couple small changes + undo the submodule change

**Comment by halter73** [comment]
@wtgodbe Is this good to merge?

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**